models/migrate.go

```diff
@@ -166,7 +166,7 @@ func InsertIssueComments(comments []*Comment) error {
 	}
 
 	for issueID := range issueIDs {
-		if _, err := db.Exec(ctx, "UPDATE issue set num_comments = (SELECT count(*) FROM comment WHERE issue_id = ?) WHERE id = ?", issueID, issueID); err != nil {
+		if _, err := db.Exec(ctx, "UPDATE issue set num_comments = (SELECT count(*) FROM comment WHERE issue_id = ? AND type=0) WHERE id = ?", issueID, issueID); err != nil {
```

Comment:
Should the `type` be quoted in case it is a SQL keyword?

And the `0` maybe can be replaced by a constant, or add a comment for it.

Reply:
> Should the type be quoted in case it is a SQL keyword?

Done.

> And the 0 maybe can be replaced by a constant, or add a comment for it.

Done.

Thanks for the quick review!


---
[source](https://lab.fedeproxy.eu/forgefriends/forgefriends/-/merge_requests/31#note_715)